### ISSUE TITLE
Ask about learning rate schedule

### ISSUE THREAD

**Issue by yjhong89** [THUDM/CogVideo#235]
Hi!

What is exact learning rate schedule as training iteration goes ?
- I am currently trying SFT, so set learning rate as 1e-5.
- Seems learning rate scheduler AnnealingLR of SAT module, but tricky to find out since some arguments are missing especially `decay_style`, `decay ratio` and `warmup_iter`
  - https://github.com/THUDM/SwissArmyTransformer/blob/b68c4460b9fa2b5312be49a1da152986c6351262/sat/training/learning_rates.py#L24 

- I checked that until 100 iteration, it is fixed to 5e-7 (default auto_warmup_steps (100), auto_warmup_rate (0.05))
- After 100 iteration, it starts warmup (learning rate increase) and then annealed (lr decrease). 
- But, I couldn't figure out how warmup_iteration is specified in SAT module.

**Comment by yjhong89** [comment]
- It seems that after 5e-7 fixed auto warm up 100 steps, annealing starts to the end of iteration.
- Then, how does scheduling learning rate effect to training result ? 1e-5 and Annealing LR is an empirical result ?

**Comment by yjhong89** [comment]
- Is there any way changing fixed learning rate scheduler by config ?? 
- Or do I need to change manually SwissArmyTransformer package ??
